src/components/DefinitionList/components/GroupLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import {Text} from '@gravity-ui/uikit';
+
+import {b} from '../utils';
+
+interface GroupLabelProps {
+    label: string;
+}
+
+export function GroupLabel({label}: GroupLabelProps) {
+    return (
+        <div className={b('subheader')}>
```

Comment:
Maybe name it `title` or `header`? `subheader` implies there is another heading level above, but there isn't